src/streaming/TextTracks.js

```diff
@@ -423,17 +426,11 @@ function TextTracks() {
                 };
             }
             else if (currentItem.type === 'html') {
+                log('Create Cue id:' + currentItem.cueID + ' start : ' + currentItem.start + ' end : ' + currentItem.end + ' timeOffset : ' + timeOffset);
                 cue = new Cue(currentItem.start - timeOffset, currentItem.end - timeOffset, '');
                 cue.cueHTMLElement = currentItem.cueHTMLElement;
-                cue.regions = currentItem.regions;
-                cue.regionID = currentItem.regionID;
+                cue.isd = currentItem.isd;
                 cue.cueID = currentItem.cueID;
-                cue.videoWidth = currentItem.videoWidth;
-                cue.videoHeight = currentItem.videoHeight;
-                cue.cellResolution = currentItem.cellResolution;
```

Comment:
cellResolution is used at https://github.com/Dash-Industry-Forum/dash.js/blob/development/src/streaming/TextTracks.js#L295 when the video element is resized. Now this atttibute has been removed, that method either needs updating or, more likely, removing since it is no longer relevant. Currently, this causes an uncaught TypeError.